package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "devDependencies": {
+    "@rollup/plugin-alias": "^3.1.9",
```

Comment:
Because these dependencies are defined at the root, they are dev dependencies of all the packages that are part of this project.